sources/slack/__init__.py

```diff
@@ -58,16 +63,28 @@ def channels_resource() -> Iterable[TDataItem]:
 
     yield channels_resource
 
-    msg_dt_fields = [
-        "ts",
-        "thread_ts",
-        "latest_reply",
-        "blocks.thread_ts",
-        "blocks.latest_reply",
-        "attachment.thread_ts",
-        "attachment.latest_reply",
-        "edited.ts",
-    ]
+    @dlt.resource(name="users", primary_key="id", write_disposition="replace")
+    def users_resource(
+        datetime_fields: List[str] = DEFAULT_DATETIME_FIELDS,
```

Comment:
It is a bad idea to pass lists as default parameters. I wonder why linter is not complaining. (maybe because this is inner function). Anyway - I do not think you need to pass it here. Just remove the argument and use `DEFAULT_DATETIME_FIELDS` like in other places

Reply:
does not look like resolved - do not pass a list as default argument. remove argument altogether. you do not really need that